.github/workflows/auto-nightly-ci.yaml

```diff
@@ -147,6 +147,9 @@ jobs:
     if: ${{ needs.get_ref.outputs.e2e_enabled == 'true' && needs.get_ref.outputs.ipfamily_dual_e2e == 'true' }}
     uses: ./.github/workflows/e2e-init.yaml
     with:
+      # github ci: openvswitch failed to run ubuntu-latest(24.04)
+      # https://github.com/spidernet-io/spiderpool/issues/4397
+      os: ubuntu-22.04
```

Comment:
24.04  不是刚出来的发型版本，为什么之前 能跑 
我们后续有一些测试 高内核的 场景，包括 生产 用 ubuntu ，这个 是一个需要解决的问题 

```
Attempt 1 to install openvswitch on spiderpool1215201358-control-plane...
debconf: delaying package configuration, since apt-utils is not installed
Succeed to install openvswitch on spiderpool1215201358-control-plane
A dependency job for openvswitch-switch.service failed. See 'journalctl -xe' for details.
```

Reply:
1. 从 https://github.com/actions/runner-images?tab=readme-ov-file 看， ubuntu24.04 似乎是12.15 号推出，也对应CI 开始挂的那天
2. 终究是要解决这问题，目前我也无法解答为什么无法运行 ovs，这 pr 只是一个临时的方案，否则所有 pr ，nighty ci 全部失败

Reply:
3. 拉 vagrant vm 可能并不能发现这问题，之前正常的 runner:

```
=========Current os version=================
ubuntu-latest
=========Current system kernel=================
6.5.0-1025-azure
```

现在出问题的 os：

```
=========Current os version=================
ubuntu-latest
=========Current system kernel=================
6.8.0-1017-azure
```

Reply:
即使找到原因，我们又能做什么？也还是只能临时回退规避 
---- 这个想法太消极了，我希望还是有调研、知道始末 才下结论

Reply:
前阵子的 github 上的 ubuntu 2404都是可用的，12.15 号的 ubuntu-latest 更新之后有这问题，这不属于我们的软件 bug ，所以暂时不想花费太多时间精力，其他还有更重要的 pr 被 pending 了, 并不是想消极对待，我认为这个问题根因的优先级不高，希望把其他 pr 解决了再回头来看这问题